tuna/example/example_lib.py

```diff
@@ -98,25 +101,24 @@ def launch_worker(self, gpu_idx, f_vals, worker_lst):
     worker_lst.append(worker)
     return True
 
-  def compose_worker_list(self, machines):
+  def compose_worker_list(self, machines) -> Optional[List[str]]:
     # pylint: disable=too-many-branches
     """! Helper function to compose worker_list
-      @param res DB query return item containg available machines
-      @param args The command line arguments
+      @param machines containg available machines
     """
-    worker_lst = []
+    worker_lst: List[str] = []
     for machine in machines:
       if self.args.restart_machine:
         machine.restart_server(wait=False)
         continue
 
       #determine number of processes by compute capacity
       # pylint: disable=duplicate-code
-      worker_ids = super().get_num_procs(machine)
+      worker_ids: range = super().get_num_procs(machine)
```

Comment:
A range would require a set of numbers separated by a stride.
get_num_procs wouldn't guarantee this, so a list would be more appropriate.